BuildTasks/InstallExtension/task.json

```diff
@@ -39,64 +39,85 @@
   "instanceNameFormat": "Install Extension",
   "inputs": [
     {
-        "name": "connectedServiceName",
-        "type": "connectedService:VstsMarketplacePublishing",
-        "label": "Marketplace connection",
-        "required": true,
-        "helpMarkDown": "Service endpoint connection to install the extension."
+      "name": "connectTo",

```

Comment:
Did you try to install an extension in TFS Local gallery using tfx-cli? What does it need to be passed in `accounts` parameter?